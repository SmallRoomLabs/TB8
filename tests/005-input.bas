10 print "Input first number:";
20 input A
30 print "Input second number:";
40 input B
50 if A = B then print "The numbers are the same"
60 if A > B then print "The first number is largest"
70 if A < B then print "The second number is largest"
75 print ""
80 goto 10
