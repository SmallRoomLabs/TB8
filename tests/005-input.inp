5555
5555
6666
7777
9999
8888
