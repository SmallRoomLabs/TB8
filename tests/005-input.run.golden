>RUN
Input first number:5555
Input second number:5555
The numbers are the same

Input first number:6666
Input second number:7777
The second number is largest

Input first number:9999
Input second number:8888
The first number is largest

Input first number:
